app/controllers/regulatorsAndDocuments/CharityScottishRegulatorRegistrationNumberController.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.regulatorsAndDocuments
+
+import config.FrontendAppConfig
+import controllers.LocalBaseController
+import controllers.actions._
+import forms.regulatorsAndDocuments.CharityScottishRegulatorRegistrationNumberFormProvider
+import javax.inject.Inject
+import models.Mode
+import navigation.RegulatorsAndDocumentsNavigator
+import pages.Section2Page
+import pages.regulatorsAndDocuments.CharityScottishRegulatorRegistrationNumberPage
+import play.api.mvc._
+import repositories.UserAnswerRepository
+import views.html.regulatorsAndDocuments.CharityScottishRegulatorRegistrationNumberView
+
+import scala.concurrent.Future
+
+class CharityScottishRegulatorRegistrationNumberController @Inject()(
```

Comment:
File name is too long, can we trim like ScottishRegulatorRegNumber and it applies to all files

Reply:
changed the file name